eZ/Publish/Core/settings/storage_engines/legacy/field_value_converters.yml

```diff
@@ -33,6 +33,11 @@ services:
         class: "%ezpublish.persistence.legacy.field_value_converter.registry.class%"
         lazy: true
 
+    ezpublish.fieldType.ezgeneric.converter:
```

Comment:
Service names should have lowercase letters, though in this case FQCN also applies.